### PR TITLE
Adapt Unlocks Endpoint

### LINKED ISSUE #4635: Adapt Unlocks Endpoint
### Description

Fetch the list of unlocks by address (voter)

### Acceptance Criteria

- Update the HTTP & RPC endpoint
- Update MSW mocks
- Response has been changed please update the usage
- Adopt unit test

### Additional Information

- HTTP GET /api/v3/pos/unlocks (prev: /api/v3/dpos/unlocks)
- RPC  get.pos.unlocks (prev: get.dpos.unlocks)
- [Response details](https://docs.google.com/document/d/1RoH4KZGJIfiUyXHkyeEwyCTAVpT-07P1OVJnKYfxbrU/edit#heading=h.44mucvmml4vu)

### PR DESCRIPTION
**Pull request by reyraa**
### What was the problem?

This PR resolves #4635

### How was it solved?

- Update endpoint URL and method name
- Update mock responses
- Update components and hooks to match to the new response schema

### How was it tested?

- Update unit tests

### CODE REVIEW

File: src/modules/account/hooks/useDeprecatedAccount.js [hunk 1]
```diff
@@ -117,7 +117,7 @@ export const useDeprecatedAccount = (accountInfo) => {
           consecutiveMissedBlocks: delegate?.consecutiveMissedBlocks,
           lastForgedHeight: delegate?.lastGeneratedHeight,
           isBanned: delegate?.isBanned,
-          totalVotesReceived: delegate?.totalVotesReceived,
+          totalStakeReceived: delegate?.totalStakeReceived,
```
Comment (soroushm) [inline src/modules/account/hooks/useDeprecatedAccount.js]: delegate -> validator

File: src/modules/wallet/components/stakes/stakeRow.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ const StakeRow = ({ data, onRowClick, accounts }) => {
         <div className={`${grid['col-sm-2']} ${styles.flexLeftAlign}`} onClick={onClick}>
           <span>
             <TokenAmount
-              val={account?.dpos.delegate.totalVotesReceived ?? 0}
+              val={account?.dpos.delegate.totalStakeReceived ?? 0}
```
Comment (soroushm) [inline src/modules/wallet/components/stakes/stakeRow.js]: its still accound.dpos or account.pos?

File: src/modules/search/hooks/useSearch.js [hunk 3]
```diff
@@ -1,48 +1,65 @@
-import { regex } from 'src/const/regex';
```
Comment (oskarleonard) [inline src/modules/search/hooks/useSearch.js]: Are we reverting back? @soroushm  recently changed this file from using useCustomQuery to using specific hooks https://github.com/LiskHQ/lisk-desktop/pull/4666

File: src/modules/wallet/components/stakes/stakeRow.js (hunk 2 above)
Comment (reyraa) [reply src/modules/wallet/components/stakes/stakeRow.js]: Fixed.

File: src/modules/account/hooks/useDeprecatedAccount.js (hunk 1 above)
Comment (reyraa) [reply src/modules/account/hooks/useDeprecatedAccount.js]: Fixed.

File: src/modules/search/hooks/useSearch.js (hunk 3 above)
Comment (reyraa) [reply src/modules/search/hooks/useSearch.js]: Done

**Review by oskarleonard — DISMISSED** [review]

File: src/modules/pos/validator/components/EditStake/index.js [hunk 4]
```diff
@@ -58,14 +58,14 @@ const EditStake = ({ history, stakeEdited, network, voting, stakesRetrieved }) =
   const [isForm, setIsForm] = useState(true);
 
   const [address] = selectSearchParamValue(history.location.search, ['address']);
-  const delegateAddress = address || currentAddress; // this holds the address of either other validators or the user's address
+  const validatorAddress = address || currentAddress; // this holds the address of either other validators or the user's address
 
-  const { data: delegates, isLoading: isLoadingDelegates } = useValidators({
-    config: { params: { address: delegateAddress } },
+  const { data: validators, isLoading: isLoadingDelegates } = useValidators({
```
Comment (ikem-legend) [inline src/modules/pos/validator/components/EditStake/index.js]: @reyraa The `isLoading` property should be renamed accordingly

File: src/modules/pos/validator/components/EditStake/index.test.js [hunk 5]
```diff
@@ -160,15 +160,15 @@ describe('EditStake', () => {
...
   it('should render the edit stake modal', async () => {
-    const delegate = mockValidators.data[0];
+    const validator = mockValidators.data[0];
 
     useSentStakes.mockReturnValue({
       data: {
         ...mockSentStakes,
         data: {
           ...mockSentStakes.data,
           votes: mockSentStakes.data.votes.map((vote, index) =>
```
Comment (ikem-legend) [inline src/modules/pos/validator/components/EditStake/index.test.js:141]: @reyraa Assuming the data from the endpoint has been updated too, shouldn't this be `stakes: ...`?

Comment (ManuGowda) [reply src/modules/pos/validator/components/EditStake/index.test.js:141]: The mocks have not been updated. so for now we have to keep it this way.

**Review by ManuGowda — APPROVED** [review]

**Review by oskarleonard — APPROVED** [review]

**Pull request merged**